### PR TITLE
Setup ModelAdmin class for each model in Team app solved.

### PR DESCRIPTION
**Pull request by RishabhJain2018**
#69 #71  solved. @deshraj @TroJan Please review the PR.

### CODE REVIEW

**Comment by TroJan** [comment]
@RishabhJain2018 : you need to rebase your branch from master, squash all the commits into a single atomic commit with a very meaningful message and the force push on this branch.

File: apps/participants/admin.py [hunk 1]
```diff
@@ -1,3 +1,17 @@
 from django.contrib import admin
+from base.admin import TimeStampedAdmin
+from .models import ParticipantStatus, Participant
 
-# Register your models here.
+
+class ParticipantStatusAdmin(TimeStampedAdmin):
```
Comment (RishabhJain2018) [inline apps/participants/admin.py]: @TroJan  Yeah Sure.

File: apps/participants/admin.py [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+    list_display = ('status',)
+
+
+class ParticipantAdmin(TimeStampedAdmin):
```
Comment (RishabhJain2018) [inline apps/participants/admin.py]: okk

**Comment by RishabhJain2018** [comment]
@TroJan I will open a new PR with all the changes. For now I am closing this. :)

**Pull request closed without merging**